Space-based file storage report endpoint

max-zilla opened the pull request:
/api/reports/storage/spaces/:id endpoint with file records for user-based storage tracking

<img width="1437" alt="Screen Shot 2020-10-21 at 8 56 30 AM" src="https://user-images.githubusercontent.com/14166971/96730865-3fab5e80-137c-11eb-8a90-4510983a7377.png">

lmarini reviewed (approved):
Looks good. Two minor comments:

- The column `space_owners` shows `0` for all resources and the column `space_admins` is empty.
- I don't see updates to the swagger documentation

max-zilla commented:
@lmarini I updated Swagger and fixed a bug where space and role was stored capitalized "Admin" instead of "admin". However I could not replicate space owners as 0, do you actually have an owner ID for the space in question in mongo? It should at minimum include the space.creator ID for the space you requested in the report.

lmarini commented:
> @lmarini I updated Swagger and fixed a bug where space and role was stored capitalized "Admin" instead of "admin". However I could not replicate space owners as 0, do you actually have an owner ID for the space in question in mongo? It should at minimum include the space.creator ID for the space you requested in the report.

Thank you. Both are fixed. One was due to a bad document in mongodb.
